com.unity.render-pipelines.universal/Runtime/2D/Passes/Utility/Light2DLookupTexture.cs

```diff
@@ -14,10 +14,14 @@ public static Texture GetLightLookupTexture()
             return s_PointLightLookupTexture;
         }
 
-        public static Texture GetFalloffLookupTexture()
+        public static Texture GetFalloffLookupTexture(Renderer2DData data)
```

Comment:
shouldn't this be the implementation of the getter of `data.fallOffLookup`?

Reply:
s_FalloffLookupTexture = data.fallOffLookup;

And like you mentioned in the last PR, if it fails to load the texture, the fallback to create Texture2D is here.

Reply:
If you don't need the fallback, no need to access data.fallOffLookup via GetFalloffLookupTexture(). I can remove this function and access it directly wherever its been used. Let me know.

Reply:
yes, this pattern is really weird... let's make it so that it is more common...  and we also get to remove yet another static... which is good.

Reply:
Got it. Will do it..

Reply:
Fixed in https://github.com/Unity-Technologies/Graphics/pull/1990/commits/a49ac4e7e04c1266bf38fccbc3dbd5e750941e7d